src/components/token/TokenBalances.js

```diff
@@ -311,19 +331,18 @@ class TokenBalances extends React.Component {
               </p>
             )
           }
-          {
-            !this.state.tokenBalanceInformationError && (
-              <>
-                <p><b>Total number of addresses:</b> { helpers.renderValue(this.state.addressesCount, true) }</p>
-                <p><b>Total number of transactions:</b> { helpers.renderValue(this.state.transactionsCount, true) }</p>
-              </>
-            )
-          }
-          {
-            this.state.tokenBalanceInformationError && (
-              <ErrorMessageWithIcon message='Error loading token balance information. Please try again.' />
-            )
-          }
+
+          {!this.state.tokenBalanceInformationError && (
+            <p><b>Total number of addresses:</b> { helpers.renderValue(this.state.addressesCount, true) }</p>
+          )}
+
+          {!this.state.tokensApiError && (
+            <p><b>Total number of transactions:</b> { helpers.renderValue(this.state.transactionsCount, true) }</p>
+          )}
+
+          {(this.state.tokensApiError || this.state.tokenBalanceInformationError) &&(
+            <ErrorMessageWithIcon message='Error loading the complete token balance information. Please try again.' />
```

Comment:
We could, it would be specific for the hathor token though (so is this error message)

I would just suggest opening an issue for this instead of doing it in this PR so we can merge this and fix the bug in production